src/renderer.ts

```diff
@@ -277,6 +279,7 @@ export class Renderer {
         ? new Map(JSON.parse(customHeaders))
         : new Map(),
       content: result,
+      varnishCache: response.headers()['x-varnish'].toLowerCase() || 'unknown',
```

Comment:
I'd make this as safe as possible
```suggestion
      varnishCache: response.headers()?.['x-varnish']?.toLowerCase() || 'unknown',
```